source/includes/_08-additional-features.md.erb

```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
         An object uniquely identifying this candidate from this provider,
         structured as it is in <a href="#data-structure">the PassFort API</a>
       </td>
+    </tr>
     <tr>
-  </tbody>
+      <td><code>addresses</code></td>
+      <td>
+        A list of CompanyAddress objects
+      </td>
+      <td>No</td>
+      <td>
+        Addresses of the company, with each CompanyAddress object structured
+        as it is in <a href="#data-structure">the PassFort API</a>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>contact</code></td>
+      <td>
+        A CompanyContactDetails object
+      </td>
+      <td>No</td>
+      <td>
+        Contact details for the company, in an object structured as it
+        is in <a href="#data-structure">the PassFort API</a>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>incorporation_date</code></td>
+      <td>
+        string
+      </td>
+      <td>No</td>
+      <td>
+        Exact date in YYYY-MM-DD format
+      </td>
+    </tr>
+    <tr>
+      <td><code>structure_type</code></td>
+      <td>
+        Object containing structured fields
+      </td>
+      <td>No</td>
+      <td>
+        Object structured the same as the `structured_company_type` field of
+        the `CompanyMetadata` object in <a href="#data-structure">the PassFort
+        API</a>: "Structured fields indicating the kind of liability its
+        members are exposed to, the ownership structure, and whether it is
+        publically traded."
```

Comment:
Is it `publicly`?

Reply:
Yes; good catch. I had just copied this text from the main PassFort API, so we should fix it there too.

Reply:
Fixed.